### ISSUE TITLE
Thaumic Pants

### ISSUE THREAD

**Issue by Dayyer1** [KryptonCaptain/Thaumic-Dyes#7]
Have you noticed the lack of special pants in Thaumcraft? There are the Goggles of Revealing, the Thaumostatic Harness, and the Boots of the Traveler, but no special pants. I suppose thaumium leggings could go in that place, but that doesn't seem quite right. They should have some sort of special ability, though. Unfortunately, I'm not sure. Maybe higher protection or higher vis discount. I thought that it was worth bringing up, I think it'd fit in this mod.
Also, Thaumostatic Harness upgrades, like fuel efficiency and protection.
Just suggestions.
I have finals in two weeks, but after that, I'll try to help with creating the mod, if that's ok.

**Comment by KryptonCaptain** [comment]
Might look at reviving the Goliath Leggings from early TT

**Comment by 63502** [comment]
Goliath Leggings? I don't recall those at all. What were they exactly? And what TC/TT version were they from? They certainly sound interesting.

**Comment by KryptonCaptain** [comment]
Not sure, found them while scrolling back TT's commit history. They're really early on

**Comment by 63502** [comment]
Ok, interesting. I just googled them, and apparently they were from TT for TC3. The wiki doesn't say anything about them other than that they give a vis discount.

**Comment by KryptonCaptain** [comment]
They have some special damage reduction traits

**Comment by 63502** [comment]
I just found the original thaumonomicon entry for TT1 and it's recipe on the FTB wiki.

> Making use of the Thaumaturge's Robes you created, you futher worked on them, creating new leggings. These leggings protect the wearer from all incoming damage, by reducing it by 1.
Futhermore, when the wearer would die by a crushing blow when they have more than 2.5 hearts, they stay barely alive with half a heart.
They also give a more powerful vis reduction.

![goliath s leggings feed the beast wiki](https://cloud.githubusercontent.com/assets/15267053/26484238/f61c6bca-41be-11e7-9b49-53547b08601f.png)

This looks like a really cool idea to implement, but it seems a bit out of place without a set to go with it.

**Comment by Dayyer1** [comment]
Yeah, seems kind of lame/pointless if it's just by itself... (I like the idea. I think that just 1 point of damage isn't a lot.) The point here is to add JUST leggings though. Maybe enhance these a bit?
What about pants that let you fall slowly if the thaumostatic harness runs out of fuel?

I'm imagining wearing all 4 pieces at once, and figuring out what that last ability could be.

Could you infuse it with a potion or buff? Or maybe a debuff is occasionally applied to mobs/players around you if you take damage.

**Comment by Dayyer1** [comment]
(Possible runic shielding upgrade, too? But that would make the leggings less worth making...)

**Comment by Dayyer1** [comment]
Double jump?

**Comment by Shahelion** [comment]
I could see the Goliath Leggings being repurposed as some sort of defensive armor set, where it reduces damage taken such that you can't take more than X damage in a single hit, with more pieces giving a lower number (with just the leggings, all hits above 4 hearts get reduced to 4 hearts, with the full set, all hits get reduced to 1 heart of damage unless already lower).

The tricky thing is, most stuff that could have been leggings has already been added as various baubles, especially belts.
